public/js/app.js

```diff
@@ -1,223 +1,241 @@
+      };
+
+      // Post message
+      cb.__call("statuses_update", params, function(reply, rate, err) {
         // Error handling
-        console.error(cloudinaryErr);
+        if (err) {
+          console.log(err);
+        } else {
+          // call postNewProject and pass imageUrl and tweetID as args
+          postNewProject(imageUrl, reply.id_str);
+        }
       });
     }
-  });
-});
 
-// When user clicks upvote or downvote button
-$(".vote-btn").on("click", function() {
-  // Set type of vote
-  var voteType;
-  if($(this).hasClass("upvote-btn")) {
-    voteType = "up";
-  } else if ($(this).hasClass("downvote-btn")) {
-    voteType = "down";
-  }
-
-  // Set API route based on ID
-  var apiRoute = `/api/issues/${$(this).attr("data-id")}`;
-
-  // AJAX request to get current score and perform appropriate PUT request based on voteType
-  // i.e. either increment or decrement current score
-  $.ajax(apiRoute, {
-    type: "GET"
-  }).then(function(res) {
-    var newScore;
-    // if voteType is up
-    if (voteType === "up") {
-      newScore = res.score + 1;
-    } else if (voteType === "down") {
-      newScore = res.score - 1;
+    // Function to post new project to Spiffy API/database
+    // Takes image URL and twitter ID as argument
+    function postNewProject(imgUrl, twitterID) {
+      var newProject = {
+        title: $("#userProjectName").val().trim(),
+        location: $("#user-location").val().trim(),
+        projectType: $("#userProjectType").val().trim(),
+        imglocation: imgUrl,
+        tweetID: twitterID,
+        score: 0,
+        userName: userName,
+        userEmail: userEmail
+      }
+
+      console.log(newProject);
+
+      $.ajax("/api/issues", {
+        data: newProject,
+        type: "POST"
+      }).then(function() {
+        console.log("new project added");
+      });
+
+      // Reload page
+      location.reload();
     }
-    // AJAX PUT request to update score
-    $.ajax(apiRoute, {
-      data: {
-        score: newScore
-      },
-      type: "PUT"
-    }).then(function(updateResponse) {
-      // Set element with class issue-score and matching data-id to score from response
-      $(`.issue-score[data-id="${updateResponse.id}"]`).text(updateResponse.score);
+
+    $("#viewOne").on("click", function() {
+      var user = "hillary";
+      $(".issue").empty();
+      $.ajax("/api/issues/" + user, {
+        type: "GET"
+      }).then(function(data) {
+        console.log(data);
+        console.log(data.issue[0].title);
+        console.log(data.issue.length);
+        for (i = 0; i < data.issue.length; i++) {
+          var newIssueDiv = $('<div class="issue">').html(
+            '<p class="issue-title">' + data.issue[i].title +
+            '</p><p class="issue-type">CATEGORY: ' + data.issue[i].projectType +
+            '</p><p class="issue-location issue-item">LOCATION: ' + data.issue[i].projectType +
```

Comment:
This is my issue - should be data.issue[i].location not data.issue[i].projectType